vendor/licenses/grammar/language-graphql.txt

```diff
@@ -0,0 +1,7 @@
+---
+type: grammar
+name: language-graphql
+license: mit
+curated: true
+---
+MIT
```

Comment:
This should have attribution (`Copyright 2016 Author McName`) and the full text of the license.